src/api/app/views/layouts/webui/responsive_ux/_login_modal.html.haml

```diff
@@ -0,0 +1,8 @@
+.modal.fade.access-modal#log-in-modal{ tabindex: -1, role: 'dialog', aria: { hidden: true } }
+  .modal-dialog.modal-dialog-centered{ role: 'document' }
+    .modal-content
+      .modal-body
+        %i.close.fas.fa-times{ data: { dismiss: 'modal' } }
+        = link_to(root_path, class: 'd-block text-center', alt: 'Logo') do
+          = image_tag('favicon.svg', height: '60')
+        = render partial: 'layouts/webui/responsive_ux/log_in_form'
```

Comment:
Shouldn't this be `= render partial: "layouts/#{responsive_namespace}/log_in_form`?

Reply:
In this case, it is not necessary, because it is a new partial that will be only accessible on `beta`. So we don't need to switch between `webui` and `webui/responsive_ux`. Not like, for example, `navigation` where need to switch.

Reply:
Ah good catch!